src/pages/adt-interpreters/regexp.md

```diff
@@ -231,7 +233,8 @@ def matches(input: String): Boolean = {
     regexp match {
       case Append(left, right) =>
         loop(left, idx).flatMap(i => loop(right, i))
-      case OrElse(first, second) => loop(first, idx).orElse(loop(second, idx))
+      case OrElse(first, second) => 
+        loop(first, idx).orElse(loop(second, idx))
```

Comment:
This line of code will be automatically wrapped in the PDF, making the code less readable.